### ISSUE TITLE
Browserify

### ISSUE THREAD

**Issue by rstacruz** [tj/frontend-boilerplate#59]
I know this repo represents your personal preferences, but I'm curious on why the choice of webpack over browserify.
- browserify can do hot module replacement via browserify-hmr and it's [pretty easy](https://github.com/substack/react-starter-hmr/blob/master/package.json#L9). no need for an external dev server like webpack-dev-server.
- postcss is supported via [browserify-postcss](https://www.npmjs.com/package/browserify-postcss), but why bother adding your css into your js files when they can be compiled as .css files?
- vendor.bundle.js can be done via [factor-bundle](https://www.npmjs.org/package/factor-bundle)
- babel is supported via [babelify](https://www.npmjs.com/package/babelify)

using browserify would probably reduce 60 lines of webpack.config.js into just ~10 lines inside package.json, and is much simpler to boot.

**Comment by tj** [comment]
I'm cool with it if the hmr stuff is as fast, I've had issues with browserify being slow in the past but that was pre-hmr. I don't really care what builder it is as long as it supports the stuff I want to use, at a glance it seemed to have less adoption these days.

**Comment by rstacruz** [comment]
Ahh.. Sounds like watchify will fix that for you :)

**Comment by tj** [comment]
Yeah maybe, seems snappier now than it was when I tried last, at least for the tiny demo app in there

**Comment by tj** [comment]
Is it possible to use postcss without streams? Can't stand node streams, I just want non-js config ideally

**Comment by rstacruz** [comment]
You mean from the cli?

> browserify -t [ browserify-postcss --plugin cssnext ] -t browserify-css

Or expressing its equivalent in the package.json "browserify" field.
Disclaimer: haven't actually tried it, so it probably doesn't work as is.
Haha

**Comment by tj** [comment]
cool I'll try it out later on my stuff and see how it goes
